### PR TITLE
discord widget

### PR DESCRIPTION
**Pull request by powercasgamer**

### CODE REVIEW

File: src/pages/index.astro [hunk 1]
```diff
@@ -51,11 +51,16 @@ import {COMMUNITY_INVITE_URL, GITHUB_ORG_URL, REDDIT_URL} from '../consts';
         </a> or contribute to our community projects on
         <a href={GITHUB_ORG_URL}
            target="_blank"
-           title="Check out our subreddit">
+           title="Check out our github">
             GitHub
         </a>!
     </p>
-    <img src="/invite.png" alt="Search in the documentation">
+    <br style="margin-top: 0.5px"/>
```
Comment (Turbotailz) [inline src/pages/index.astro]: Inline styles should be avoided at all costs. Could you please put this in the style sheet below?

File: .gitignore [hunk 2]
```diff
@@ -5,14 +5,14 @@ dist/
 
 # dependencies
 node_modules/
+pnpm-lock.yaml
```
Comment (Turbotailz) [inline .gitignore]: It's not really a good idea to ignore lock files, but this project uses NPM, not PNPM so it doesn't really matter too much.

When installing you should be doing so with `npm install`

**Review by Turbotailz — CHANGES_REQUESTED** [review]
Thanks! Just a couple things.

**Comment by Turbotailz** [comment]
Also, because Cloudflare pages doesn't support PR previews from external repositories, could you please add a screenshot showing what it looks like?

**Comment by powercasgamer** [comment]
> Also, because Cloudflare pages doesn't support PR previews from external repositories, could you please add a screenshot showing what it looks like?

firefox_9A0VzyQMf6

**Pull request merged**